youtube_dlc/downloader/external.py

```diff
@@ -118,6 +118,41 @@ def _call_downloader(self, tmpfilename, info_dict):
         _, stderr = process_communicate_or_kill(p)
         if p.returncode != 0:
             self.to_stderr(stderr.decode('utf-8', 'replace'))
+
+        if 'url_list' in info_dict:
+            file_list = []
+            for [i, url] in enumerate(info_dict['url_list']):
+                tmpsegmentname = '%s_%s.frag' % (tmpfilename, i)
+                file_list.append(tmpsegmentname)
+            with open(tmpfilename, 'wb') as dest:
```

Comment:
Ideally this should obey `--continue`/`--no-overwrite`, but we can ignore it for now if implementing it is too complicated